sp800-63a/sec5_proofing.md

```diff
@@ -142,7 +142,7 @@ Supervised remote identity proofing and enrollment transactions SHALL meet the f
 4. The CSP SHALL require that all digital verification of evidence (e.g., via chip or wireless technologies) be performed by integrated scanners and sensors.
 5. The CSP SHALL require operators to have undergone a training program to detect potential fraud and to properly perform a virtual in-process proofing session.
 6. The CSP SHALL employ physical tamper detection and resistance features appropriate for the environment in which it is located. For example, a kiosk located in a restricted area or one where it is monitored by a trusted individual requires less tamper detection than one that is located in a semi-public area such as the concourse of a shopping mall.
-7. The CSP SHALL ensure that all communications take place over a mutually-authenticated encrypted session.
+7. The CSP SHALL ensure that all communications take place over an authenticated protected channel.
```

Comment:
In this case you probably do want the remote proofing station to be authenticated as well, so
an authenticated protected channel -> a mutually authenticated protected channel